arctic/store/version_store.py

```diff
@@ -21,6 +22,7 @@
 
 VERSION_STORE_TYPE = 'VersionStore'
 _TYPE_HANDLERS = []
+AVOID_FALLBACK_HANDLERS = bool(os.environ.get('AVOID_FALLBACK_HANDLERS'))
```

Comment:
Is it worth making this a property of the library/store rather than a global property?